supervision/geometry/utils.py

```diff
@@ -10,9 +10,8 @@ def get_polygon_center(polygon: np.ndarray) -> Point:
     This function takes in a polygon as a 2-dimensional numpy ndarray and
     returns the center of the polygon as a Point object.
 
-    The center is calculated as center of frame.
-    polygon -> polygon, where p[i] = p[i + 1] + p[i] / 2,
-     with mass = length of vector p[i + 1] - p[i]
+    Еhe center is calculated as the center
```

Comment:
Typo "Ehe"

Reply:
> Typo "Ehe"

sorry, I was in a hurry :)

Regarding your comments, I thought about it, and yes, you are right, it can easily happen that in the polygon of the same object there will be a different number of points on different frames of the video, I also agree that it is better to use the “solid” algorithm, since it is universal, but it doesn’t lose much in speed

(I solved the problem with the condition in this algorithm, if the sum is equal to zero, then the algorithm returns the average of all points, this is a sufficient approximation, since this condition can only be satisfied if there are very few points or they form a straight line)

and I wanted to write to you in private messages because I have a couple of ideas related to the 'geometry' module, but not with this function specifically